Godeps/Godeps.json

```diff
@@ -442,47 +437,47 @@
 		},
 		{
 			"ImportPath": "github.com/containerd/containerd/api/services/containers/v1",
-			"Comment": "v1.0.0-beta.2-159-g27d450a0",
+			"Comment": "v1.0.0-beta.2-159-g27d450a",
```

Comment:
Why are we bumping all of these too? Are those really transitive dependencies of etcd client?

Reply:
I just followed the guide [Using godep to manage dependencies](https://github.com/kubernetes/community/blob/master/contributors/devel/godep.md) and it updated godeps for me. I'm not sure why this was updated. But maybe it's a mistake: `g27d450a0` vs `g27d450a`. I will revert it manually.

Reply:
Took another look and found that this is just a Comment change. I guess it's a upstream Comment change. Here it doesn't bump the real code.

Reply:
See also: https://github.com/kubernetes/kubernetes/pull/57811#discussion_r159820267